conformance/conformance_test_runner.cc

```diff
@@ -307,7 +307,8 @@ void ForkPipeRunner::SpawnTestProgram() {
 }
 
 void ForkPipeRunner::CheckedWrite(int fd, const void *buf, size_t len) {
-  if (write(fd, buf, len) != len) {
```

Comment:
static_cast?

Reply:
@TeBoring , I have updated the change as per your comments. Please review.

Reply:
should this be `static_cast<size_t>`?

Reply:
Yes. Updated the changes. Thanks for review.

Reply:
@TeBoring Please review the change, and help to merge this